src/pages/p/[projectId]/index.tsx

```diff
@@ -17,13 +18,16 @@ export default function Page({
   metadata,
 }: InferGetStaticPropsType<typeof getStaticProps>) {
   const pid = projectId ? BigInt(projectId) : 0n
+  // Sanitize before slicing to prevent partial HTML tags at the end.
+  const sanitizedDescription = metadata?.description
```

Comment:
Can we move this to the SEO component so we get the functionality on all SEO descriptions?

Reply:
Not needed, as project pages were the only page with HTML in their descriptions (others are hard-coded to typical strings).